conventions/code-review.md

```diff
@@ -0,0 +1,57 @@
+# Code Reviews
+
+## Introduction to Code Reviews
+
+This section details the steps involved in reviewing code within a submitted pull request. Refer to the separate [pull request](pull-request.md) process document for information on submitting PRs.
+
+## PR Assignment
+
+Upon PR submission assign the code review the team member who is working solo for that day as per the calender. If you are working solo assign to another member of the team and ensure to rotate assignees.
+
+## Review and Feedback
+
+### 1. Code Clarity
+
+- Well-structured and easy to understand code.
+- Meaningful variable and function names that reflect their purpose.
+
+### 2. Implementation
+
+- Does the new code do what it was designed to do? 
+- Does it reflect the requirements of the linked issue? 
+- Does it handle potential edge cases correctly?
+
+### 3. Style and consistency
+
+- Does it follow a consistent style and pattern within the project?
+- Ensure all code is pep8 compliant, Refer to [PEP 8](https://pep8.org/) guidelines for details.
+
+### 4. Maintainability
+
+-  How easy it is to modify, integrate, or extend with the existing codebase in the future.
+
+### 5. Documentation
+
+- Ensure all functions are documented using [docs-and-comments](docs-and-comments.md) as reference.
+
+### 6. Testing
+
+- All Python code should be thoroughly tested.
+- Test coverage should exceed 90%.
```

Comment:
How should we measure coverage? Should we use a coverage package?

Reply:
I believe in our Makefile the coverage package is included so this should run within our ci/cd pipeline also